include/seqan3/alphabet/quality/all.hpp

```diff
@@ -34,12 +34,76 @@
 
 #pragma once
 
-#include <seqan3/alphabet/quality/aliases.hpp>
-#include <seqan3/alphabet/quality/quality_composition.hpp>
-#include <seqan3/alphabet/quality/concept.hpp>
-#include <seqan3/alphabet/quality/illumina18.hpp>
-
 /*!\file
  * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
  * \brief Meta-header that includes all headers from alphabet/quality/
  */
+
+#include <seqan3/alphabet/quality/aliases.hpp>
+#include <seqan3/alphabet/quality/concept.hpp>
+#include <seqan3/alphabet/quality/qualified.hpp>
+#include <seqan3/alphabet/quality/phred42.hpp>
+#include <seqan3/alphabet/quality/phred63.hpp>
+#include <seqan3/alphabet/quality/phred68.hpp>
+
+/*!\defgroup quality Quality
+ * \brief Contains the various quality score types.
+ * \ingroup alphabet
+ *
+ * \par Introduction
+ *
+ * Quality score sequences are usually output together with the DNA (or RNA) sequence
+ * by sequencing machines like the Illumina Genome Analyzer. The quality score
+ * of a nucleotide is also known as Phred score and is an integer score being
+ * inversely proportional to the propability p that a base call is <B>incorrect</B>.
+ * Which basically means that the higher a Phred score is, the higher the probabality that the
+ * corresponding nucleotide is correct for that position. There exists mainly two
+ * variants of its computation:
+ *   1. Salinger format with Q = -10log_10(p)
+ *   2. Solexa format with Q = -10log_10(p/(1-p))
+ * Thus, for very low quality levels the scores vary significantly and need to be
+ * corrected by an offset before being compared. The Phred score range is not
+ * displayable with one digit, and is therefore mapped to ASCII values. Depending
+ * on the format and the analyzer machine generation, the mappings may differ.
+ * Output files containing DNA sequences and their quality scores are usually
+ * stored in the .fastq or .fq format.
+ * This sub-module offers multiple quality alphabets that can be used with regular
+ * containers and ranges.
+ *
+ * \par Encoding Schemes
+ *
+ * | Format                      | Quality Type | Phred Score Range  | Rank Range   | ASCII Range  |
+ * |:---------------------------:|:-------------|:------------------:|:------------:|:------------:|
+ * | Sanger, Illumina 1.8+ short | phred42      | [0 .. 41]          | [0 .. 41]    | ['!' .. 'J'] |
+ * | Sanger, Illumina 1.8+ long  | phred63      | [0 .. 62]          | [0 .. 62]    | ['!' .. '~'] |
+ * | Solexa, Illumina [1.0; 1.8[ | phred68      | [-5 .. 62]         | [0 .. 67]    | [';' .. '~'] |
+ *
+ * The most distributed format is the Sanger and Illumina 1.8+ format. Despite
+ * typical phred scores for Illumina machines range from 0 to maximal 41, it is
+ * possible that processed reads reach higher scores. If 41 is not exceeded it is
+ * recommended to use phred42 to due to its smaller space consumption compared
+ * to phred63.
+ * For other formats, like Solexa and Illumina 1.0 to 1.7 the type phred68 is
```

Comment:
Did you mean to rename phred68 to phred68legacy, because phred42 and phred63 represent the same encodings, but the latter one allows to store higher values, whereas phred68 maps to a different ASCII range?

Reply:
Yes, it also clearly states that all the formats that phred68 is designed for are actually deprecated!